apps/website/src/content/docs/flex.mdx

```diff
@@ -1,20 +1,70 @@
 ---
 title: Flex
-description: A utility component to work with CSS flexbox and iTwinUI design tokens.
+description: A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens. Supports all flex properties. Can be used with or without `Flex.Item` and `Flex.Spacer` subcomponents.
 thumbnail: #TODO
 group: utilities
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='flex--basic' />
+## Usage
+
+Wrap `Flex` around the components you want to be the children of the flex container. To set a certain universal space between the children, pass iTwinUI size tokens (e.g. `x`, `xs`) into the `gap` prop. The `gap` prop is set to be `xs` by default.
 
 <LiveExample src='Flex.main.jsx'>
   <AllExamples.FlexMainExample client:load />
 </LiveExample>
 
-A utility component that makes it easier to work with CSS flexbox and iTwinUI design tokens. Supports all flex properties. Can be used with or without `Flex.Item` and `Flex.Spacer` subcomponents.
+### Flex properties
+
+The `Flex` container offers properties which reflect similar behaviors to the CSS flexbox, such as `alignItem`, `justifyContent`, `flexDirection`, and `flexWrap`.
+
+#### `justifyContent`
+
+<LiveExample src='Flex.justifyContent.jsx'>
+  <AllExamples.FlexJustifyContentExample client:load />
+</LiveExample>
+
+#### `alignItems`
+
+<LiveExample src='Flex.alignItems.jsx'>
+  <AllExamples.FlexAlignItemsExample client:load />
+</LiveExample>
+
+#### `flexDirection`
+
+<LiveExample src='Flex.direction.jsx'>
+  <AllExamples.FlexDirectionExample client:load />
+</LiveExample>
+
+#### `flexWrap`
+
+<LiveExample src='Flex.wrap.jsx'>
+  <AllExamples.FlexWrapExample client:load />
+</LiveExample>
+
+### Spacer
+
+The `Flex.Spacer` subcomponent fills up the available space between flex items. This subcomponent is useful when it is necessary to push certain items to the edges.
+
+<LiveExample src='Flex.spacer.jsx'>
+  <AllExamples.FlexSpacerExample client:load />
+</LiveExample>
+
+### Flex Items
+
+This `Flex.Item` allows customizing flex items through the `flex`, `alignSelf` and `gapBefore`/`gapAfter` props.
+
+<LiveExample src='Flex.flexItems.jsx'>
+  <AllExamples.FlexWithFlexItemsExample client:load />
+</LiveExample>
+
+The `gapBefore`/`gapAfter` props can used to override the gap at an individual level, for when you need a different gap than the one set in the parent `Flex`.
+
+<LiveExample src='Flex.individualGaps.jsx'>
+  <AllExamples.FlexWithIndividualGapsExample client:load />
+</LiveExample>
 
-## Props
+### Props
```

Comment:
This should continue to be h2. It's not a subheading of "Usage".

Reply:
I have changed this to be h2.